torchtune/training/quantization.py

```diff
@@ -142,15 +153,20 @@ def get_quantizer_mode(quantizer: Optional[Callable]) -> Optional[str]:
     Currently supported:
 
     - :class:`~torchao.quantization.quant_api.Int8DynActInt4WeightQuantizer`: "8da4w" (requires ``torch>=2.3.0``)
-    - :class:`~torchao.quantization.prototype.qat.Int8DynActInt4WeightQATQuantizer`: "8da4w-qat" (requires ``torch>=2.4.0``)
+    - :class:`~torchao.quantization.qat.Int8DynActInt4WeightQATQuantizer`: "8da4w-qat" (requires ``torch>=2.4.0``)
```

Comment:
I think we can fully remove both of these torch requirements as we assume stable PyTorch (i.e. >= 2.5)